Miniforge/construct.yaml

```diff
@@ -1,16 +1,16 @@
-name: Miniforge3
+name: Miniforge
+version: 4.7.11-0
 
 channels:
   - https://conda.anaconda.org/conda-forge
-  - https://conda.anaconda.org/anaconda
   # This will continue to be a dependency until the compilers are
-  # moved to conda-forge or to anaconda
+  # moved to conda-forge.
   - https://conda.anaconda.org/c4aarch64   [linux-aarch64]
 
 write_condarc: True
 license_file: ../LICENSE
 
 specs:
   - python >=3,<3.8
```

Comment:
Can we specify a version here (`3.7`) so that the conda solver doesn't change it.

Reply:
Agree. Should we also specify it in `version`? Or do we just manually match the python version that is in the conda version we use?

Reply:
@hmaarrfk, thoughts?

Reply:
I would prefer to match the Python version defined in conda.

Reply:
> do we just manually match the python version that is in the conda version we use?

What do you mean by this?

Reply:
`conda 4.7.11` comes with a specific Python version I guess? I think we should just match tis Python version and so we don't set the Python version in the miniforge version string because that information is already contained in the conda version we are using.

Reply:
> conda 4.7.11 comes with a specific Python version I guess?

No. It's built for python 2.7,3.6,3.7 and 3.8

Reply:
Ok, then we need to specify that version in miniforge version.

Reply:
`miniforge37-4.7.11-0.sh`?

Reply:
On the miniconda home page there is only one Python 3 version used (`3.7`). Why not following this?

`Miniforge3-4.7.11-0.sh`

Reply:
I mark resolve @isuruf. Feel free to re-open the discussion if you have concerns.